projects/schematics/src/migrations/2_0/constructor-deprecations/data/cart-details.component.migration.ts

```diff
@@ -137,4 +137,82 @@ export const CART_DETAILS_COMPONENT_MIGRATIONS: ConstructorDeprecation[] = [
       },
     ],
   },
+  {
+    class: CART_DETAILS_COMPONENT,
```

Comment:
@znikola would you mind evaluating this file? I'm unsure if the different migration scripts are what we should end up with?